app/views/users/index.html.erb

```diff
@@ -1,12 +1,12 @@
 <p>Here is a list of all users</p>
 <% @users.each do |user| %>
     <div>
-    <p><%= user.name %></p>
-    </div>
-    <div>
-    <p><%= user.bio %></p>
+    <img src= "<%= user.photo %>" />
     </div>
     <div>
-    <p><%= user.photo %></p>
+    <%= link_to user_path(user.id) do %>
+    <p><%= user.name %></p>
+    <% end %>
+    <p>Number of Posts: <%= user.posts_counter %></p>
```

Comment:
- [ ] Kindly ensure that the users page matches the given [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).
- [ ] Navigating to `http://[::1]:3000/users/1/posts/1/` to get post of the first post of the first user is not rendering correctly. Kindly ensure to improve upon the same.

**Your Users view:**
![image](https://user-images.githubusercontent.com/12745474/199690596-40cf9119-3812-47c5-bab8-05a5dd257c2b.png)

**Users view (wireframe):**
![image](https://user-images.githubusercontent.com/12745474/199690901-7e503606-f5ed-4a77-a3c2-ec7256f3795e.png)

**Users post error:**

![image](https://user-images.githubusercontent.com/12745474/199691177-b6be43ae-df80-422f-b8f1-b384395114e5.png)